onnxruntime/core/providers/coreml/builders/helper.cc

```diff
@@ -84,16 +84,10 @@ bool IsInputSupported(const Node& node, const NodeArg& input,
       return false;
     }
 
-    if (dim == 0) {
-      if (node.OpType() == "Resize" && &input == node.InputDefs()[1]) {
```

Comment:
To be really correct this also needed to allow an empty input for `scales` if opset was 11. That was getting too convoluted for here so added a simple flag so an op builder implementation can allow empty inputs in this basic check and handle them internally.